src/checkbox-group/checkbox-group__header.jsx

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+const debug = require("debug")("Leeruniek:LUCheckboxSeparator")
+
+import * as React from "react"
+import cx from "classnames"
+
+import css from "./checkbox-group.css"
+
+import type { LUCheckboxGroupHeaderType } from "./checkbox-group.types"
+
+export const LUCheckboxGroupHeader = (
```

Comment:
Use `ComponentStateType` and `ComponentPropsType` when defining state and props types.